src/ngraph/pass/constant_folding.cpp

```diff
@@ -193,16 +234,33 @@ void ngraph::pass::ConstantFolding::construct_constant_reshape()
 
 template <class T>
 shared_ptr<op::Constant> make_constant_broadcast(shared_ptr<op::Constant> constant,
-                                                 shared_ptr<op::Broadcast> broadcast)
+                                                 shared_ptr<op::Broadcast> broadcast,
+                                                 ngraph::BuildCFMap& m_cfmap)
 {
     auto out_shape = broadcast->get_shape();
     vector<T> out_vec(shape_size(out_shape));
 
-    runtime::reference::broadcast<T>(constant->get_vector<T>().data(),
-                                     out_vec.data(),
-                                     constant->get_shape(),
-                                     out_shape,
-                                     broadcast->get_broadcast_axes());
+    auto handler = m_cfmap.find(type_index(typeid(ngraph::op::Broadcast)));
+    if (handler != m_cfmap.end())
+    {
+        vector<T> in_vec = constant->get_vector<T>();
```

Comment:
Constant stores the data in an array of T internally. Use `constant->get_data_ptr<T>()` to get a pointer to the raw data. This will save a copy.